compliance/management/commands/decrypt_exports_inquiry.py

```diff
@@ -0,0 +1,76 @@
+"""
+Management command to decrypts a user's ExportInquiryLog record
+"""
+# import sys
+
+from django.core.management import BaseCommand
+from django.contrib.auth import get_user_model
+from nacl.encoding import Base64Encoder
+from nacl.public import PrivateKey
+
+from compliance.api import decrypt_exports_inquiry
+
+User = get_user_model()
+
+
+class Command(BaseCommand):
+    """
+    Management command to decrypts a user's ExportInquiryLog record
+    """
+
+    help = "Decrypts a user's ExportInquiryLog record"
+
+    def add_arguments(self, parser):
+        """
+        Definition of arguments this command accepts
+        """
+        group = parser.add_mutually_exclusive_group(required=True)
+        group.add_argument("--id")
+        group.add_argument("--email")
+        group.add_argument("--username")
+
+    def handle(self, *args, **options):
+        """Run the command"""
+
+        if options["id"]:
+            user = User.objects.get(id=options["id"])
+        elif options["username"]:
+            user = User.objects.get(username=options["username"])
+        elif options["email"]:
+            user = User.objects.get(email=options["email"])
+
+        if user is None:
+            self.stderr.write(self.style.ERROR("User doesn't exist "))
+            # sys.exit(1)
+
+        log = user.exports_inquiries.last()
+
+        if log is None:
+            self.stderr.write(self.style.ERROR("User has no ExportsInquiryLog records"))
+            # sys.exit(2)
```

Comment:
Why is this and the other `sys.exit` above commented out?

Reply:
Ah good catch, that was me trying to figure out why pylint was going bonkers